.github/workflows/project.yml

```diff
@@ -0,0 +1,16 @@
+name: Add pull requests to review project
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add Pull Request to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/containerd/projects/5
+          github-token: ${{ secrets.GHA_ADD_TO_PROJECT }}
```

Comment:
Do we need to have a separate token for this? Wont the default `GITHUB_TOKEN` work?

Reply:
It wasn't working, but there are still a few more combinations I could try